lib/interpreters/split_clustering.rb

```diff
@@ -608,7 +608,7 @@ def build_service_like(shipment, activity)
                             quantities: shipment.quantities)
       end
 
-      def build_services_from_shipments(shipments)
+      def build_services_from_shipments(depot_ids, shipments)
```

Comment:
It is suprising that we didn't see this bug before, this means there is no test that passes through this function. Can you add one please?